quiz.js

```diff
@@ -0,0 +1,29 @@
+var tree = {
+	height: "",
+	character: ""
+};
+var row = "";
+var space = " ";
+
+function buildTree (tree) {
+	tree.height = document.getElementById("height").value;
+	// console.log("height innerHTML:", document.getElementById("height").value);
+	tree.character = document.getElementById("character").value;
+	for (var i = 1; i <= tree.height ; i+= 1) {
+		// console.log("i:", i);
+			row += tree.character; 
+			space = " ";
+			if (i>1) {
+				row += tree.character;
+			};
```

Comment:
Lines 14-18 should be out-dented a step to match up with the other lines contained in the outermost for loop.